core/src/main/kotlin/in/specmatic/stub/HttpStub.kt

```diff
@@ -65,9 +67,21 @@ class HttpStub(
         log: (event: LogMessage) -> Unit = dontPrintToConsole
     ) : this(parseGherkinStringToFeature(gherkinData), scenarioStubs, host, port, log)
 
+    companion object {
+        const val JSON_REPORT_PATH = "./build/reports/specmatic"
+        const val JSON_REPORT_FILE_NAME = "stub_usage_report.json"
+    }
+
     private val threadSafeHttpStubs = ThreadSafeListOfStubs(_httpStubs.filter { it.stubToken == null }.toMutableList())
     private val threadSafeHttpStubQueue = ThreadSafeListOfStubs(_httpStubs.filter { it.stubToken != null }.reversed().toMutableList())
 
+    private val _logs: MutableList<StubEndpoint> = Collections.synchronizedList(ArrayList())
```

Comment:
There should be a test for this. Have say 10 threads running simultaneously making requests, and we should have 10 logs in this list.

Reply:
Sure, I have added the test.